packages/victory-tooltip/src/flyout.js

```diff
@@ -99,6 +99,7 @@ Flyout.propTypes = {
   center: PropTypes.shape({ x: PropTypes.number, y: PropTypes.number }),
   cornerRadius: PropTypes.number,
   datum: PropTypes.object,
+  disableInlineStyles: PropTypes.bool,
```

Comment:
The prop added here doesn't do anything to this primitive yet. It needs to prevent the `style` prop from being applied to the `Path` component that `Flyout` renders.

Reply:
Oh! never mind. I remembered that logic is centralized in `Helpers.evaluateStyle`